### PR TITLE
Create combased-lkmex.md

### PR DESCRIPTION
**Pull request by combased-dev**

### CODE REVIEW

File: LKMEX-whitelist/combased-lkmex.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+We are the first e-commerce project on Elrond and official Shopify partner. We provide quality blockchain services as well. Also we have our NFT collections: Combeys, Combots
+
+Purpose: 
+
+We have a transparent CDO wallet, which has LP for farming LKMEX. That LP is increased from our e-commerce and web3 development services revenue. 
+
+Address of Smart Contract:
+
+erd1qqqqqqqqqqqqqpgqss2kwutkwdvpyey60u7felml0jmrqp6hmkdqadcl5z
```
Comment (sasurobert) [inline LKMEX-whitelist/combased-lkmex.md:17]: Is this a multi sig contract ? We need a well defined code in order to whitelist.

**Review by sasurobert — CHANGES_REQUESTED** [review]

File: LKMEX-whitelist/combased-lkmex.md (hunk 1 above)
Comment (combased-dev) [reply LKMEX-whitelist/combased-lkmex.md:17]: It is not multi sig. Should it be? Ok I will attach the code (Krogan fork). Can we edit the code later?

Comment (combased-dev) [reply LKMEX-whitelist/combased-lkmex.md:17]: @sasurobert hello? :)

Comment (sasurobert) [reply LKMEX-whitelist/combased-lkmex.md:17]: We need access to the code and we need to see if this can be used for wrong purposes.

**Comment by sasurobert** [comment]
Need access to code. Please redo the PR!

**Comment by combased-dev** [comment]
@sasurobert we got advised by xdevguild and made a new PR. Please accept, when possible!
https://github.com/ElrondNetwork/elrond-specs/pull/37

**Pull request closed without merging**